Description: Debian's codec2 libraries have two new modes. Add support
 for them in the grc blocks.
Author: A. Maitland Bottoms <bottoms@debian.org>

--- a/gr-vocoder/grc/vocoder_codec2_decode_ps.xml
+++ b/gr-vocoder/grc/vocoder_codec2_decode_ps.xml
@@ -39,11 +39,19 @@
 			<name>1200 bps</name>
 			<key>codec2.MODE_1200</key>
 		</option>
+		<option>
+			<name>700 bps</name>
+			<key>codec2.MODE_700</key>
+		</option>
+		<option>
+			<name>700B bps</name>
+			<key>codec2.MODE_700B</key>
+		</option>
 	</param>
 	<sink>
 		<name>in</name>
 		<type>byte</type>
-		<vlen>{ 0: 64, 1: 48, 2: 64, 3: 56, 4: 52, 5: 48 }[$mode]</vlen>
+		<vlen>{ 0: 64, 1: 48, 2: 64, 3: 56, 4: 52, 5: 48, 6: 28, 7: 28 }[$mode]</vlen>
 	</sink>
 	<source>
 		<name>out</name>
--- a/gr-vocoder/grc/vocoder_codec2_encode_sp.xml
+++ b/gr-vocoder/grc/vocoder_codec2_encode_sp.xml
@@ -39,6 +39,14 @@
 			<name>1200 bps</name>
 			<key>codec2.MODE_1200</key>
 		</option>
+		<option>
+			<name>700 bps</name>
+			<key>codec2.MODE_700</key>
+		</option>
+		<option>
+			<name>700B bps</name>
+			<key>codec2.MODE_700B</key>
+		</option>
 	</param>
 	<sink>
 		<name>in</name>
@@ -47,7 +55,7 @@
 	<source>
 		<name>out</name>
 		<type>byte</type>
-		<vlen>{ 0: 64, 1: 48, 2: 64, 3: 56, 4: 52, 5: 48 }[$mode]</vlen>
+		<vlen>{ 0: 64, 1: 48, 2: 64, 3: 56, 4: 52, 5: 48, 6: 28, 7: 28 }[$mode]</vlen>
 	</source>
 </block>
 
